test/tests/outputs/postprocessor/tests

```diff
@@ -6,6 +6,7 @@
     type = 'Exodiff'
     input = 'postprocessor.i'
     exodiff = 'postprocessor_exodus.e postprocessor_exodus2.e'
+    capture_perf_graph = false
```

Comment:
What is the reason for disabling this for some of the cases but not all?

Reply:
This specific case sets `outputs = all` for one of the postprocessors

Reply:
Aaa I see, thanks!